### PR TITLE
averaged accuracy over multiple datasets

### LINKED ISSUE #9: Feature scaling for RidgeCVClassifier
Scikit-learn supports various feature scaling methods. We need to evaluate which scaling approach works best for our models:

- No scaling
- Min-Max scaling
- Standard scaling (mean/std)
- Robust scaling

### Tasks

1. Test all scaling methods and report model performance metrics.
2. Check whether concatenating different scaled features (e.g., Standard scaled + Robust scaled) improves results.
3. Evaluate how accuracy changes when using concatenated features with, for example, MultiROCKET features.
4. Report findings clearly, including any improvements from feature concatenation.

### Expected Outcome

- Comparison of model accuracy for each scaling method.
- Insights into whether feature concatenation provides performance gains.

### PR DESCRIPTION
**Pull request by lejkosir**
Finished testing for issue #9, results are in notebooks/mrHydra-scaler-testing.ipynb.

### CODE REVIEW

**Comment by gasperpetelin** [comment]
If possible, combine MultiRocketBenchmarkClassifier and MultiRocketHydraBenchmarkClassifier

File: notebooks/mrHydra-scaler-testing.ipynb [hunk 1]
```diff
@@ -0,0 +1,514 @@
...
+   "source": [
+    "from aeon.classification import BaseClassifier\n",
+    "from aeon.transformations.collection.convolution_based import MultiRocket\n",
+    "from sklearn.linear_model import RidgeClassifierCV\n",
+    "from sklearn.preprocessing import StandardScaler, RobustScaler, MinMaxScaler, QuantileTransformer\n",
+    "import numpy as np\n",
+    "from tscglue.models import NoScaler, SparseScaler\n",
+    "from aeon.utils.validation import check_n_jobs\n",
+    "\n",
+    "class MultiRocketBenchmarkClassifier(BaseClassifier):\n",
```
Comment (gasperpetelin) [inline notebooks/mrHydra-scaler-testing.ipynb]: If possible, merge both classes for classification into 1.

File: notebooks/mrHydra-scaler-testing.ipynb [hunk 2]
```diff
@@ -0,0 +1,514 @@
...
+   "metadata": {},
+   "cell_type": "markdown",
+   "source": "### Load results from local cache",
+   "id": "890a0ecf290a1d3"
+  },
+  {
+   "metadata": {},
+   "cell_type": "code",
+   "source": [
+    "def load_all_results(cache_dir=\"scaler-combinations\"):\n",
```
Comment (gasperpetelin) [inline notebooks/mrHydra-scaler-testing.ipynb]: Here, I think you can just use pl.read_parquet('scaler-combinations\*.parquet')

File: notebooks/mrHydra-scaler-testing.ipynb [hunk 3]
```diff
@@ -0,0 +1,514 @@
...
+   "metadata": {},
+   "cell_type": "code",
+   "source": [
+    "scaler_names = [\"sparse\", \"standard\", \"robust\", \"minmax\", \"none\", \"quantile\"]\n",
+    "combos = []\n",
+    "cache = LocalFileCache(\"scaler-combinations\")\n",
+    "for dataset in univariate:\n",
+    "    for hy in scaler_names:\n",
+    "        for mr in scaler_names:\n",
+    "            combos.append((dataset, hy, mr, f\"MR-{mr}-HY-{hy}\"))\n",
```
Comment (gasperpetelin) [inline notebooks/mrHydra-scaler-testing.ipynb]: You can use combos = list(product(univariate, combinations(scaler_names, 2))). No for loops. And the name gets constructed in a loop under.
